### DISCUSSION TITLE
Can I use Python for the server side ?

### DISCUSSION THREAD

**Discussion by NelloHo** [neutralinojs/neutralinojs#1085] [Q&A]
Hi,
Definitly using html/css/js for the front end is convinent, but can I use Python for the backend ?
as I'm used to Flask and I want to use some python library (numpy/python-chess) for my app.

**Comment by owlysk** [comment]
I don't think so. Maybe throught extensions it will be possible.
Maybe check python alternative at [reddit](https://www.reddit.com/r/Python/comments/3itauz/does_python_have_something_similar_to_electron/?xpromo_edp=enabled) or [GitHub](https://github.com/sudhakar3697/awesome-electron-alternatives)

**Comment by hschneider** [comment]
I just released a Python Extension for Neutralino:
https://github.com/hschneider/neutralino-ext-python
